### DISCUSSION TITLE
220v ac to 12v ac tranformer

### DISCUSSION THREAD

**Discussion by acamu** [MarkBryanMilligan/LanternPowerMonitor#46] [Q&A]
Hello,

Someone have found an "220v to 12v ac tranformer with a jack output" in the case of an european installation ?

The VT is needed or not ? can i use the lantern without the voltage ?

Thank for the help
Arnaud

**Comment by DaT1ck** [comment]
Posting in case anyone else has the question since you've probably figured it out already.  Yes, the 12V transformer is required for the system to work.

You can search for "12v power supply for LED strips" and get a bunch that should work for you.
